product_docs/docs/transporter/2_preview/preparing_db/preparing_postgres_source_databases.mdx

```diff
@@ -0,0 +1,129 @@
+---
+title: "Preparing Postgres source databases"
+---
+
+<div id="preparing_postgres_sources" class="registered_link"></div>
+
+Configuring Postgres for EDB Transporter requires administrative privileges. Create a change data capture (CDC) migration role with limited privileges for data migration.
+
+Execute SQL statements with psql or a similar client.
+<!--What is this command doing?-->
```

Comment:
Please clarify what the command that follows is doing in a sentence to precede the command.

Reply:
done